.github/ISSUE_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,46 @@
+<!-- 
```

Comment:
This is the template for pull request and not issue, which is not a scope of this issue.